app/src/main/java/org/simple/clinic/remoteconfig/RemoteConfigSync.kt

```diff
@@ -0,0 +1,43 @@
+package org.simple.clinic.remoteconfig
+
+import com.google.firebase.remoteconfig.FirebaseRemoteConfig
+import io.reactivex.Completable
+import io.reactivex.Single
+import org.simple.clinic.sync.BatchSize
+import org.simple.clinic.sync.ModelSync
+import org.simple.clinic.sync.SyncConfig
+import org.simple.clinic.sync.SyncGroup
+import org.simple.clinic.sync.SyncInterval
+import timber.log.Timber
+import javax.inject.Inject
+
+class RemoteConfigSync @Inject constructor(
+    private val remoteConfig: FirebaseRemoteConfig,
+    private val cacheExpiration: FirebaseRemoteConfigCacheExpiration
+) : ModelSync {
+
+  override fun sync(): Completable = pull()
+
+  override fun push(): Completable = Completable.complete()
+
+  override fun pull(): Completable {
+    return Completable.fromAction {
+      remoteConfig.fetch(cacheExpiration.value.seconds)
+          .addOnCompleteListener { task ->
+            if (task.isSuccessful) {
+              Timber.i("Firebase remote config updated successfully")
+              remoteConfig.activateFetched()
+            } else {
+              Timber.w("Failed to update Firebase remote config")
```

Comment:
The [docs](https://firebase.google.com/docs/remote-config/android#caching_and_throttling) say that if we exceed the limit for syncing, it will throw a `FirebaseRemoteConfigFetchThrottledException`. Is this case handled?